doc/2018/2018-07-31.md

```diff
@@ -0,0 +1,111 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight:
+The Ubuntu Server Installer (subiquity-based) has been released in Bionic
```

Comment:
"Along with the first point release of Ubuntu Bionic (18.04.1), the Ubuntu Server installer was updated."

Reply:
replaced.